package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "github-actions",
```

Comment:
github-actions-setup-ansible

Reply:
are we going to put new actions in different repositories or all actions in this one?

if latter, i think `github-actions` will just work fine?

Reply:
I thought it under the setup-ansible/folder. `github-actions` looks good to me then